app/javascript/react/components/LocationSearch/LocationSearch.tsx

```diff
@@ -13,11 +13,10 @@ import { useAppDispatch } from "../../store/hooks";
 import { setFetchingData } from "../../store/mapSlice";
 import { determineZoomLevel } from "../../utils/determineZoomLevel";
 import { UrlParamsTypes, useMapParams } from "../../utils/mapParamsHandler";
-import { screenSizes } from "../../utils/media";
-import useScreenSizeDetection from "../../utils/useScreenSizeDetection";
 import * as S from "./LocationSearch.style";
 
 const OK_STATUS = "OK";
+const DEFAULT_ZOOM = 13;
```

Comment:
We should ask the user how much they want to zoom in when clicking on the location browser.